### PR TITLE
[AR-40] Added Privacy Policy and Terms of Service

### PR DESCRIPTION
**Pull request by hiercomichael**
added Terms of Service and Privacy Policy Page.

### CODE REVIEW

File: components/Common/Footer/styled.ts [hunk 1]
```diff
@@ -103,6 +103,33 @@ export const LinksWrapper = styled.div`
...
+  }
+
+  @media (max-width: ${(props) => props.theme.breakpoints.md2}) {
+    left: 4rem;
+  }
+
+  @media (max-width: ${(props) => props.theme.breakpoints.md}) {
+    left: 2rem;
+  }
+`;
```
Comment (idiglove) [inline components/Common/Footer/styled.ts]: this is bad smell for code. why do we need this? cant we have just one constant style?

cant we simply align the links to the copy right from the bottom

Comment (hiercomichael) [reply components/Common/Footer/styled.ts]: @idiglove done. realized it is not needed. https://github.com/idiglove/aintboard/pull/491/commits/5f264b8cbe1fcb98f25eb096bbb738b6987f21cb

Comment (idiglove) [reply components/Common/Footer/styled.ts]: can we remove in this component too?

**Review by idiglove — COMMENTED** [review]
i have some comments in discord

**Review by idiglove — APPROVED** [review]
awesome, finally haha

**Pull request merged**